### PR TITLE
Add the doc label check

### PR DESCRIPTION
**Pull request by fengtao1998**
A script that automatically checks the document label of the pull request and runs in the pulsar community. 

Add tags according to the description of the pull request. 

When the doc tag does not exist, it will automatically reply to remind the author to improve the description

### CODE REVIEW

**Comment by fengtao1998** [comment]
@eolivelli @timmyyuan PTAL

**Comment by Anonymitaet** [comment]
Hi @eolivelli could you please help review? thanks

File: doc-label-check/.github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: CI
```
Comment (timmyyuan) [inline doc-label-check/.github/workflows/ci.yml:1]: Do we still need `.github` here?

**Review by timmyyuan — CHANGES_REQUESTED** [review]

File: doc-label-check/.github/workflows/ci.yml (hunk 1 above)
Comment (fengtao1998) [reply doc-label-check/.github/workflows/ci.yml]: Thanks. I have deleted all unnecessary files.

**Review by timmyyuan — APPROVED** [review]

**Comment by fengtao1998** [comment]
> @fengtao1998
> the index.js looks like a compiled .js file, do you have the source code ?

I've added the source code section

**Comment by Anonymitaet** [comment]
@eolivelli Tao has incorporated your comment, can you review this PR? Thanks

**Comment by fengtao1998** [comment]
@eolivelli PTAL

**Review by eolivelli — APPROVED** [review]
Lgtm

**Pull request merged**